testapi.pm

```diff
@@ -1549,6 +1571,72 @@ sub mouse_hide(;$) {
     query_isotovideo('backend_mouse_hide', {offset => $border_offset});
 }
 
+=head2 mouse_drag
+  mouse_drag([$startpoint, $endpoint, $startx, $starty, $endx, $endy, $button, $timeout]);
+
+Click mouse C<$button>, C<'left'> or C<'right'>, at a given location, hold the button and drag 
+the mouse to another location where the button is released. You can set the C<$startpoint> 
+and C<$endpoint> by passing the name of the needle tag, i.e. the mouse drag happens between
+the two needle areas. Alternatively, you can set all the coordinates explicitly with C<$startx>,
+C<$starty>, C<$endx>, and C<$endy>. You can also set one point using a needle and another one
+using coordinates.  If both the coordinates and the needle are provided, the coordinates 
+will be used to set up the locations and the needle location will be overridden.
+
+=cut
+
+sub mouse_drag {
+    my %args = @_;
+    my $startx =
+      my $starty =
+      my $endx =
+      my $endy = 0;
```

Comment:
My Perl parser strikes here.

Reply:
This was required by the Travis test (number 4) which was choking if there was not such an indentation.

Reply:
Don't refer to tests by their "numbers". I assume you refer to the tidy check. The problem here is that the lines are missing semicolons and doing an initialization like this is quite uncommon in Perl. I would rather write it as `my ($startx, $starty, $endx, $endy);`. (It looks like the default-initialization with 0 is not required here.)